benefits/core/migrations/0001_initial.py

```diff
@@ -51,6 +51,7 @@ class Migration(migrations.Migration):
                 ("start_sub_headline", models.TextField()),
                 ("start_item_heading", models.TextField()),
                 ("start_item_details", models.TextField()),
+                ("start_help_anchor", models.TextField()),
```

Comment:
NP but should be running `bin/makemigrations.sh` rather than editing these directly.

Reply:
Omg I had no idea about this (!!!)

Reply:
I rebased `dev` and ran this command.